.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Wing Development",
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+  "features": {
+    "ghcr.io/devcontainers/features/aws-cli:1": {},
+    "ghcr.io/devcontainers/features/terraform:1": {},
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {},
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": 18
+    },
+    "ghcr.io/devcontainers/features/rust:1": {},
+    "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {}
+  },
+  "mounts": [
+    "source=${localEnv:HOME}/.ssh,target=/home/vscode/.ssh,type=bind"
```

Comment:
This should normally not be necessary when the ssh keys are loaded in to the host ssh-agent, the DevContainer will try and connect to the agent sock and you'll be able to use the keys that you've added to the agent

Reply:
Thanks for the comment feedback. This would be a just in case mounting, I dont think there should be harm to include it for convenience sake.